test/integration/targets/unarchive/tasks/main.yml

```
@@ -174,6 +174,86 @@
 - name: remove our tar.gz unarchive destination
   file: path={{remote_tmp_dir}}/test-unarchive-tar-gz state=absent
 
+- name: Create a group to chown to
+  group:
+    name: testgroup
+
+- name: Create a user to chown to
+  user:
+    name: testuser
+    groups:
+    - testgroup
+
+- name: create our tar.gz unarchive destination for chown
+  file: path={{remote_tmp_dir}}/test-unarchive-tar-gz state=directory
```

Use one module parameter per line instead of key-value pairs here and for other tasks.

```suggestion
  file: 
    path: "{{remote_tmp_dir}}/test-unarchive-tar-gz"
    state: directory
```

This is the preferred style to use in test playbooks.

---

@mattclay It looks like there's a solid mix of them in this file, should I convert all the pre-existing ones to this style as well?

---

I would, but in a separate cleanup PR.